### ISSUE TITLE
Aliases can be dict (map of strings) OR list of them

### ISSUE THREAD

**Issue by vsoch** [autamus/binoc#5]
I think binoc is hiccuping on recipes that look like this:
```yaml
aliases:
  - name: run-notebook
    command: jupyter notebook --no-browser --port=$(shuf -i 2000-65000 -n 1) --ip 0.0.0.0
    options: --home ${HOME} --bind ${HOME}/.local:/home/joyvan/.local
```
which is a long handed way of writing:

```yaml
aliases:
  run-notebook: jupyter notebook --no-browser --port=$(shuf -i 2000-65000 -n 1) --ip 0.0.0.0
```
but adds options.

The struct is defined here:

https://github.com/autamus/binoc/blob/3323526ff05947170af92f366a7b3ce39f3b3ac1/parsers/shpc.go#L48

and the error looks like:

```
Parse Error: Couldn't Read --> /github/workspace/registry/jupyter/datascience-notebook/container.yaml
Error: yaml: unmarshal errors:
  line 12: cannot unmarshal !!seq into map[string]string
```
so that's my suspicion!

**Comment by vsoch** [comment]
I also think there is something going on in this function - https://github.com/autamus/binoc/blob/892e4d6463af069a134fc87a1c3e867ebc0447f4/repo/pull_request.go#L96. I had a bunch of PRs open, but a test that I added had an error so I closed them and deleted the branches. But I think the PRs are still found and this function is returning some non null value so binoc calls them "skipped."

**Comment by vsoch** [comment]
I'll stop triggering the workflow for now - I feel like I'm going to break something, lol. Probably should get a little more sleep too!

**Comment by alecbcs** [comment]
Ohh this is where this emails thread came from! Hahaha! Closing since the fix for this was implemented in commit c71c2f41c53084b8cb6299431e8d34fb2b5c0c98.
